OpenMage/magento-lts
# Remove "default" themes from 20.x

This PR removes app/design/frontend/default and skin/frontend/default.

Those themes are very old, not good looking and nobody (who keeps magento somewhat updated) is using them.

With this PR we delete almost 580 (useless) files from the repository.

### Fixed Issues
1. Fixes https://github.com/OpenMage/magento-lts/issues/1037

### Manual testing scenarios (*)
Just browse a new installation and see that the RWD theme (the default since some time) doesn't need the old "default" ones to work.

### Questions or comments
- The only people who could have problems with this PR is somebody using composer. But really, who's using composer and this extremely old themes at the same time? If they're not using composer and just copying the new version on top of the other there will be no problem cause the old files with still be there.
- I've big stores in productions without these 2 folders since years without any problems (with commercial themes or custom made ones)
